apollo-router/src/axum_http_server_factory.rs

```diff
@@ -549,6 +551,10 @@ where
 
                                 (parts, StreamBody::new(body)).into_response()
                             } else {
+                                parts.headers.insert(
+                                    "content-type",
```

Comment:
Use https://docs.rs/http/latest/http/header/constant.CONTENT_TYPE.html instead

Reply:
fixed in [273f150](https://github.com/apollographql/router/pull/1577/commits/273f1507d701bb02e8ba78d473eba661f5482244)